package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "yarn sls offline"
```

Comment:
This requiers to install sls as global

Reply:
sls is installed as a dev dependency. We should be fine.

Reply:
In order to use `sls` you need to install sls as global. So if you use `yarn sls` you do not have to

Reply:
https://stackoverflow.com/questions/44420381/how-does-npm-scripts-prioritise-local-dependency-over-global-ones

Reply:
Yes